parse/ConditionPythonParser.cpp

```diff
@@ -782,7 +820,18 @@ void RegisterGlobalsConditions(boost::python::dict& globals) {
         globals[op.first] = enum_wrapper<Condition::ContentType>(op.second);
     }
 
+    for (const auto& op : std::initializer_list<std::pair<const char*, Condition::SortingMethod>>{
+            {"MaximumNumberOf", Condition::SortingMethod::SORT_MAX},
+            {"MinimumNumberOf", Condition::SortingMethod::SORT_MIN},
+            {"ModeNumberOf",    Condition::SortingMethod::SORT_MODE},
+            {"UniqueNumberOf",  Condition::SortingMethod::SORT_UNIQUE}})
+    {
```

Comment:
No `SortingMethod::SORT_RANDOM`?

Reply:
It was added differently compared to https://github.com/freeorion/freeorion/blob/ba36a6d0f13bf3364c206eb0829e6974e2c4fa8b/parse/ConditionParser3.cpp#L235-L239